ast-nodes/expressions/operators/div-operator.dart

```diff
@@ -29,6 +31,9 @@ class DivOperator extends BinaryRelation implements Product {
   }
 
   void checkSemantics() {
-    // TODO: implement
+    leftOperand.checkSemantics();
+    rightOperand.checkSemantics();
+    leftOperand = ensureType(leftOperand, RealType());
```

Comment:
Not so simple. Since we have no integer division, I think the semantics of the div operator should be similar to C:
* if one of the operands is real, do real division
* otherwise do integer division

Also don't forget to set the `isConstant` and `resultType` fields inside the `checkSemantics` method